bedrock/firefox/templates/firefox/testflight.html

```diff
@@ -1,131 +1,138 @@
-          </div>
-
-          <label class="terms-check-label" for="id_terms">
-            <input id="id_terms" name="terms" required="required" type="checkbox">
-            <span class="title">
-            {% trans url='#terms-conditions' %}
-              I have read and agree to these
-              <a href="{{ url }}">Terms and Conditions</a>
-            {% endtrans %}
-            </span>
-          </label>
-        </div>{#-- /#terms-format-wrapper --#}
-
-        {{ newsletter_form.privacy|safe }}
-      </div>{#-- /#form-fields --#}
+          </p>
 
-      <div id="newsletter-spinner" class="hidden"></div>
-    </form>
+          <fieldset class="mzp-u-inline">
+            <legend>Format</legend>
+            <p>{{ newsletter_form.fmt|safe }}</p>
+          </fieldset>
 
-    <div id="newsletter-form-thankyou" class="thank newsletter-form newsletter-form-extra">
-      <h2>{{ _('Congrats!') }}</h2>
-
-      <h3>{{ _('You have been added to the Firefox for iOS testing program.') }}</h3>
-
-      <p>
-      {% trans %}
-        When test spots are available, you will receive an invitation from
-        TestFlight, Apple's App Store program that manages pre-release
-        software, to participate in the program.
-      {% endtrans %}
-
-        <strong>{{ _('Please accept it.') }}</strong>
-      </p>
-
-      <p>{{ _('Be aware that this email may be in your spam folder.') }}</p>
+          <p>
+            <label for="privacy" class="mzp-u-inline">
+              <input id="id_terms" name="terms" required aria-required="true" type="checkbox">
+              {% trans url='#terms-conditions' %}
+              I have read and agree to these
+              <a href="{{ url }}">Terms and Conditions</a>
+              {% endtrans %}
+            </label>
+          </p>
+          <p>
+            {{ newsletter_form.privacy|safe }}
+          </p>
+
+          <p class="mzp-c-form-submit">
+            <button class="mzp-c-button" id="footer_email_submit" type="submit">{{ _('Submit') }}</button>
+          </p>
+        </fieldset>
+
+        <div id="newsletter-spinner" class="hidden"></div>
+      </form>
+
+      <div id="newsletter-thanks" class="mzp-c-newsletter-thanks">
+        <h3>{{ _('Congrats!') }}</h3>
+        <h4>{{ _('You have been added to the Firefox for iOS testing program.') }}</h4>
+
+        <p>
+          {% trans %}
+          When test spots are available, you will receive an invitation from
+          TestFlight, Apple's App Store program that manages pre-release
+          software, to participate in the program.
+          {% endtrans %}
+          <strong>{{ _('Please accept it.') }}</strong>
+        </p>
+
+        <p>{{ _('Be aware that this email may be in your spam folder.') }}</p>
+      </div>
     </div>
-  </div>{#-- /.container --#}
-</section>{#-- /#intro --#}
+  </aside>
 
-<section id="terms-conditions">
-  <div class="container">
+  <section class="mzp-l-content" id="terms-conditions">
```

Comment:
Could this content feels a little wide for readability. Perhaps it could benefit from a slightly narrower container?